websites/A/AnimeJunkies/dist/metadata.json

```
@@ -0,0 +1,19 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "author": {
+      "name": "RedstoneZockt",
+      "id": "355333222596476930"
+    },
+    "service": "AnimeJunkies",
+    "description": {
+      "en": "Official Presence for AnimeJunkies"
+    },
```

```suggestion
    "description": {
      "en": "The community website about anime!",
      "de": "Die Communitywebseite über Anime!",
      "nl": "De communitywebsite over anime!"
    },
```
Official description